### DISCUSSION TITLE
Paperbits update frequency

### DISCUSSION THREAD

**Discussion by FXZFun** [Azure/api-management-developer-portal#2143] [General]
How often does the releases from paperbits get included into the developer portal?
There's some features that would be nice to use that are available in paperbits but not the portal, specifically the page_content widget in the layout editor.
When can I expect that functionality to be included in the portal?

**Comment by azaslonov** [comment]
@FXZFun, it gets incoroporated into this repo pretty often, but release to managed version happens on longer cadence due to larger integration cycle. This specific feature is coming in ongoing release, which should be over some time next week.

**Reply by FXZFun** [reply]
Awesome, thanks!
